README.md

```diff
@@ -1 +1,1205 @@
+| |_) | (_) || ||  ___/| | | || (_) |
+|_.__/ \__,_||_| \____/|_| |_| \__,_|
+
+
+Logging in to balena-cloud.com
+? How would you like to login? (Use arrow keys)
+❯ Web authorization (recommended)
+  Credentials
+  Authentication token
+  I don't have a balena account!
+```
+
+You will be asked how you wish to authenticate with balenaCloud. The
+easiest method is that of 'Web authorization' which will bring up a browser
+window (and ask you to first login to balenaCloud if you have not) and ask you
+to confirm you wish to login.
+
+Other authentication methods include using your username and password
+credentials or authentication token.
+
+Once logged in, a [JWT](https://jwt.io/) session token will be saved in your
+home directory (`~/.balena/token`). Be aware that the lifetime of a balena
+JWT is limited to seven days, after which time reauthentication will be
+required.
+
+## 2. Creating an Application and Provisioning a Device
+
+### 2.1 Creating an Application
+
+Applications can be created via the dashboard or via the balenaCLI. We're going
+to create a new application via balenaCLI called `cliApp`. Run the following
+command:
+```
+$ balena app create cliApp
+```
+This will ask you which device type you wish to create the application for.
+You can scroll up and down this list using the arrow keys. For now, exit
+the command by hitting `Ctrl-C`, as there's another, non-interactive way to
+do this which we'll use instead. Type:
+```
+$ balena devices supported
+```
+to see a list of all supported device types by balenaCloud. For the rest of
+this masterclass we're going to assume you're using a balenaFin, but you can
+just as easily use any supported balena device.
+We'll pass the balenaFin device type (`fincm3`) to the application creation
+command directly:
+```
+$ balena app create cliApp --type fincm3
+Application created: cliApp (fincm3, id 1234567)
+```
+As can be seen, this will return the name of the application, its type
+(`fincm3`) and its unique ID. Obviously, if you're using a different device
+type, pass the appropriate device type to the `app create` command instead.
+
+Non-interactive commands are useful when you need to script actions via
+balenaCLI for a shellscript (although balena also includes HTTPS endpoints and
+SDKs which can be used for this purpose).
+
+You can list the applications currently owned by (or shared with) your account
+by typing:
+```
+$ balena apps
+ID      APP NAME         DEVICE TYPE      ONLINE DEVICES DEVICE COUNT
+1234567 cliApp           fincm3           0              0
+```
+
+### 2.2 Provisioning a Device
+
+You can now provision your balenaFin by downloading a provisioning image from
+the balenaCloud dashboard. Be sure to download a development image, as we'll
+be utilising its features later.
+
+Once the provisioning image is downloaded, connect your balenaFin to your development machine and run
+[Etcher to flash it](https://www.balena.io/fin/1.1/docs/getting-started/#Flashing-the-OS).
```

Comment:
It might be worth linking to etcher directly, for the non-fin users (there's almost certainly going to be more pi3 users, at least for the time being).